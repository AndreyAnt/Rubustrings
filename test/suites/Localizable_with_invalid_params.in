"NO_ARGUMENT" = "Invalid format %@";
"MIXED_%@%d%@%d" = "Some %@ %d %@ with incorrect %@ arguments";
"MIXED_POSITION_1_%@%d" = "Both %1$@ and %2$@";
"MIXED_POSITION_2_%@%d" = "Both %2$@ and %1$d";
"MISSING_PARAM_%@%@%@" = "Have %1$@ and %3$@ but missing 2";
"MISSING_PARAM_2_%@%@%@" = "Have %1$@ and %2$@ but missing 3";
"NOT_ALL_SPECIFIED_%@%@%@" = "There are %1$@ and %2$@ but no number for %@";
"POSITION_IN_KEY_1_%1$@_%@" = "%@ %@";
"POSITION_IN_KEY_2_%@_%1$@" = "%@ %@";
"WRONG_LENGTH_%d" = "Short %hd";
