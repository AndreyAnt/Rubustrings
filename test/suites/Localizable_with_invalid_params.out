[34mProcessing file: "test/suites/Localizable_with_invalid_params.in"[0m
test/suites/Localizable_with_invalid_params.in:1: error: variables mismatch: "NO_ARGUMENT" = "Invalid format %@";
test/suites/Localizable_with_invalid_params.in:2: error: variables mismatch: "MIXED_%@%d%@%d" = "Some %@ %d %@ with incorrect %@ arguments";
test/suites/Localizable_with_invalid_params.in:3: error: variables mismatch: "MIXED_POSITION_1_%@%d" = "Both %1$@ and %2$@";
test/suites/Localizable_with_invalid_params.in:4: error: variables mismatch: "MIXED_POSITION_2_%@%d" = "Both %2$@ and %1$d";
test/suites/Localizable_with_invalid_params.in:5: error: variables mismatch: "MISSING_PARAM_%@%@%@" = "Have %1$@ and %3$@ but missing 2";
test/suites/Localizable_with_invalid_params.in:6: error: variables mismatch: "MISSING_PARAM_2_%@%@%@" = "Have %1$@ and %2$@ but missing 3";
test/suites/Localizable_with_invalid_params.in:7: error: variables mismatch: "NOT_ALL_SPECIFIED_%@%@%@" = "There are %1$@ and %2$@ but no number for %@";
test/suites/Localizable_with_invalid_params.in:8: error: variables mismatch: "POSITION_IN_KEY_1_%1$@_%@" = "%@ %@";
test/suites/Localizable_with_invalid_params.in:9: error: variables mismatch: "POSITION_IN_KEY_2_%@_%1$@" = "%@ %@";
test/suites/Localizable_with_invalid_params.in:10: error: variables mismatch: "WRONG_LENGTH_%d" = "Short %hd";
[31m[1m
Result: âœ˜ Some errors detected[0m[0m
